sdk/src/beta9/abstractions/base/runner.py

```diff
@@ -399,7 +399,13 @@ def create_tmp_jupyter_file(input_history: List[str]) -> tempfile.NamedTemporary
     for code in input_history:
         if isinstance(code, list):
             code = "".join(code)
-        tmp_file.write(code + "\n\n")
+
+        # Skip command lines
+        for line in code.split("\n"):
+            if line.startswith("get_ipython"):
```

Comment:
I'm no jupyter expert, but are there more commands we need to consider? If so, we might need to maintain a list of them and change how we exclude them here.

Reply:
I ran a little test of the various other magic jupyter things and it seems like they all start with the get_ipython call, so we should be covered. 

![Screenshot 2024-09-13 at 15 55 49](https://github.com/user-attachments/assets/366034ed-b6b5-41e7-8aba-dd1f62b2aea1)